projects/angular-image-viewer/src/lib/angular-image-viewer.component.ts

```diff
@@ -205,7 +205,11 @@ export class AngularImageViewerComponent implements OnInit, OnChanges {
   }
 
   private canNavigate(event: any) {
-    return event == null || (this.config.allowKeyboardNavigation && this.hovered);
+    if (event.type === 'keyup') {
```

Comment:
I can see event == null check below. So I suspect we should have null check in if and else condition before type check.